db/primary_migrate/20220727112113_add_threatmetrix_to_proofing_components.rb

```diff
@@ -0,0 +1,8 @@
+class AddThreatmetrixToProofingComponents < ActiveRecord::Migration[6.1]
+  def change
+    add_column :proofing_components, :threatmetrix, :boolean
+    add_column :proofing_components, :threatmetrix_review_status, :string
+    add_column :proofing_components, :threatmetrix_risk_rating, :string
+    add_column :proofing_components, :threatmetrix_policy_score, :string
```

Comment:
our other columns on the table are more generic? Did we consider things like `device_fingerprinting_vendor: "threatmetrix"` or things like that?

Reply:
I thought about something more generic like risk assessment vendor.  However, it sounded to me like there are multiple vendors weighing in on that risk score behind the scenes with our different data points.   I do like the idea of identifying the specific device fingerprinting vendor since there will probably generally ever be just one so I can add that in.